darts/models/torch_forecasting_model.py

```diff
@@ -151,7 +156,7 @@ def __init__(self,
                  lr_scheduler_cls: torch.optim.lr_scheduler._LRScheduler = None,
                  lr_scheduler_kwargs: Optional[Dict] = None,
                  loss_fn: nn.modules.loss._Loss = nn.MSELoss(),
-                 model_name: str = "torch_model_run",  # TODO uid
+                 model_name: str = "torch_model_run",  # TODO: uid
```

Comment:
Just realized this hadn't been done when working on `TorchForecastingModel`. I'll add it to the backlog, since right now having multiple `TorchForecastingModel` with the default name would create conflicting checkpoints…